libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -373,13 +357,10 @@ void AP_MotorsHeli_Dual::calculate_scalars()
 
     _collective2_zero_thrst_pct = _collective_zero_thrust_pct;
 
-    // configure swashplate 1 and update scalars
-    _swashplate1.configure();
-    _swashplate1.calculate_roll_pitch_collective_factors();
-
-    // configure swashplate 2 and update scalars
-    _swashplate2.configure();
```

Comment:
I have moved configure() out of calculate scalars and into init.  It does mean that we now need a reboot for the param values to take effect, but I have updated the effected parameter meta data to reflect this.  We shouldn't be constantly calling configure, its wasteful.